src/azul/service/elasticsearch_service.py

```diff
@@ -537,21 +537,80 @@ def transform_summary(self,
                 'sum',
                 field='contents.cell_suspensions.total_estimated_cells_'
             )
-            # Add a total cell count aggregate
+            # FIXME: Remove deprecated field totalCellCount
+            #        https://github.com/DataBiosphere/azul/issues/3650
+            # Add a cell suspensions cell count aggregate
             es_search.aggs.metric('totalCellCount',
                                   'sum',
                                   field='contents.cell_suspensions.total_estimated_cells_')
+            # Add a cell suspensions cell count aggregate from projects
+            # that have no project level estimated cell count.
+            field = 'contents.projects.estimated_cell_count'
+            field_type = self.field_type(catalog, tuple(field.split('.')))
+            null_value = field_type.to_index(None)
+            es_search.aggs.bucket(
+                'withoutProjectCellCount',
+                'filter',
+                filter=Q('terms', **{field: [0, null_value]})
+            ).bucket(
+                'totalCellCount',
+                'sum',
+                field='contents.cell_suspensions.total_estimated_cells_'
+            )
+            # Add a cell suspensions cell count aggregate from projects
+            # that have some project level estimated cell count.
+            es_search.aggs.bucket(
+                'withProjectCellCount',
+                'filter',
+                filter=Q('bool',
+                         must_not=[Q('terms', **{field: [0, null_value]})])
```

Comment:
Instead of two aggs with complementary filters, use one `filters` aggregation in conjunction with `other_bucket` (https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-aggregations-bucket-filters-aggregation.html#other-bucket).